net.sourceforge.fuse_emulator.Fuse.yaml

```diff
@@ -0,0 +1,123 @@
+  - /share/man
+modules:
+  # SDL 1.2 compatibility layer for SDL 2
+  - shared-modules/SDL/sdl12-compat.json
+  # Audiofile is used for loading Spectrum data from WAV files
+  - name: libaudiofile
+    config-opts:
+      - --disable-docs
+      - --disable-examples
+      - --disable-flac
+      - --disable-static
+    sources:
+      - type: archive
+        url: http://audiofile.68k.org/audiofile-0.3.6.tar.gz
+        sha256: cdc60df19ab08bfe55344395739bb08f50fc15c92da3962fac334d3bff116965
+      - type: patch
+        path: audiofile-build.patch
+    cleanup:
+      - /bin
+  # Library to handle Spectrum snapshots, tapes and other files
+  - name: libspectrum
+    sources:
+      - type: archive
+        url: https://netcologne.dl.sourceforge.net/project/fuse-emulator/libspectrum/1.5.0/libspectrum-1.5.0.tar.gz
+        sha256: a353cb46e9b1a281061d816353ea010d0a6fe78e6a17aa0b7b74271ca5e4acfc
+  # Misc utilities (not needed if you simply want to run Fuse)
+  - name: fuse-utils
+    sources:
+      - type: archive
+        url: https://netcologne.dl.sourceforge.net/project/fuse-emulator/fuse-utils/1.4.3/fuse-utils-1.4.3.tar.gz
+        sha256: 0d2588bc37fcc6b039010c9ed68aa1e0bf513a80978737b5545f1d4f1684c325
+  # The emulator itself
+  - name: fuse
+    config-opts:
+      - --with-x
+      - --with-audio-driver=sdl
+      - --enable-desktop-integration
+    rm-configure: true
+    sources:
+      - type: archive
+        url: https://netcologne.dl.sourceforge.net/project/fuse-emulator/fuse/1.6.0/fuse-1.6.0.tar.gz
+        sha256: 3a8fedf2ffe947c571561bac55a59adad4c59338f74e449b7e7a67d9ca047096
+      - type: patch
+        paths:
+          - fuse-config-dir.patch
+          - set-generic-icon.patch
+      - type: file
+        path: net.sourceforge.fuse_emulator.Fuse.appdata.xml
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -f -i -I m4
+      - type: shell
+        commands:
+          - install -m644 -D net.sourceforge.fuse_emulator.Fuse.appdata.xml /app/share/appdata/net.sourceforge.fuse_emulator.Fuse.appdata.xml
+  # The SDL version of the emulator
+  - name: fuse-sdl
+    config-opts:
+      - --with-sdl
+      - --with-audio-driver=sdl
+      - --disable-desktop-integration
+    make-args:
+      - EXEEXT=-sdl
+    make-install-args:
+      - EXEEXT=-sdl
+    rm-configure: true
+    sources:
+      - type: archive
+        url: https://netcologne.dl.sourceforge.net/project/fuse-emulator/fuse/1.6.0/fuse-1.6.0.tar.gz
+        sha256: 3a8fedf2ffe947c571561bac55a59adad4c59338f74e449b7e7a67d9ca047096
+      - type: patch
+        paths:
+          - fuse-config-dir.patch
+          - set-generic-icon.patch
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -f -i -I m4
+      - type: shell
+        commands:
+          - cp /app/share/applications/fuse.desktop /app/share/applications/net.sourceforge.fuse_emulator.Fuse.SDL.desktop
```

Comment:
you should use `rename-desktop-file` in the top level of the manifest instead.

Reply:
oh nevermind there are two desktop files.

Reply:
There are two `.desktop` files because I build the app twice with two different UIs. The first (main) desktop file is renamed with `rename-desktop-file`, but I don't know how to rename the second one without doing it by hand.